testData/solutions/win32.json5

```diff
@@ -14,13 +14,64 @@
                 "options": {
                     "filename": "${{environment}.APPDATA}\\Freedom Scientific\\JAWS\\17.0\\Settings\\enu\\DEFAULT.JCF"
                 },
-                "supportedSettings": {
-                    "Voice Profiles.ActiveVoiceProfileName": {},
-                    "options.SayAllIndicateCaps": {},
-                    "options.TypingEcho": {},
-                    "options.SayAllMode": {},
-                    "Braille.BrailleMode": {},
-                    "options.SayAllIgnoreShiftKeys": {}
+                "supportedSettings": { // Full description of settings in JAWS: https://github.com/gloob/gpii-settings/blob/master/AT-Tools/JAWS/Default.jcf.csv
+                    "Voice Profiles.ActiveVoiceProfileName": { // I don't think this is something that the user should change since we always store the voice settings into our own profile name "GPII"
+                        "schema": {
```

Comment:
do we still want all these schemas after our last talk, or..?

Reply:
We can at least leave them around representing Javi's research. I've added a comment explaining that they will never be supported in this form